query/src/provider.rs

```diff
@@ -476,15 +478,20 @@ impl Deduplicater {
             }
 
             // Go over non_duplicates_chunks, build a plan for it
-            let mut non_duplicate_plans = Self::build_plans_for_non_duplicates_chunks(
-                self.ctx.child_ctx("build_plans_for_non_duplicates_chunks"),
-                Arc::clone(&table_name),
-                Arc::clone(&output_schema),
-                self.no_duplicates_chunks.to_vec(),
-                predicate,
-                output_sort_key.as_ref(),
-            )?;
-            plans.append(&mut non_duplicate_plans);
+            if !self.no_duplicates_chunks.is_empty() {
+                trace!(
+                    "Build one scan node for the rest of neither-duplicated-nor-ovelapped chunks."
```

Comment:
Typo in this trace message, could affect searching for particular logs:

```suggestion
                    "Build one scan node for the rest of neither-duplicated-nor-overlapped chunks."
```